samples/embeddings/python/local.settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+      "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+      "AzureWebJobsFeatureFlags": "EnableWorkerIndexing",
+      "FUNCTIONS_WORKER_RUNTIME": "python",
+      "AZURE_OPENAI_ENDPOINT": "https://<resource-name>.openai.azure.com/",
+      "EMBEDDING_MODEL_DEPLOYMENT_NAME": "embedding"
```

Comment:
the value should be same as in csharp-ooproc sample (default model for embedding as a placeholder)

Reply:
Updated